HPXMLtoOpenStudio/resources/EPvalidator.xml

```diff
@@ -446,6 +446,18 @@
       <sch:assert role='ERROR' test='number(h:AnnualHeatingEfficiency[h:Units="Percent"]/h:Value) &lt;= 1 or not(h:AnnualHeatingEfficiency[h:Units="Percent"]/h:Value)'>Expected AnnualHeatingEfficiency[Units="Percent"]/Value to be less than or equal to 1</sch:assert>
     </sch:rule>
   </sch:pattern>
+
+  <sch:pattern>
+    <sch:title>[HeatingSystemType=PackagedTerminalAirConditionerHeating]</sch:title>
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/h:HeatingSystem[h:HeatingSystemType/h:PackagedTerminalAirConditionerHeating]'>
+      <sch:assert role='ERROR' test='count(h:DistributionSystem) = 0'>Expected 0 element(s) for xpath: DistributionSystem</sch:assert>
+      <sch:assert role='ERROR' test='count(h:HeatingSystemFuel) = 1'>Expected 1 element(s) for xpath: HeatingSystemFuel</sch:assert>
+      <sch:assert role='ERROR' test='h:HeatingSystemFuel[text()="electricity"] or not(h:HeatingSystemFuel)'>Expected HeatingSystemFuel to be 'electricity'</sch:assert>
+      <sch:assert role='ERROR' test='count(h:HeatingCapacity) &lt;= 1'>Expected 0 or 1 element(s) for xpath: HeatingCapacity</sch:assert>
+      <sch:assert role='ERROR' test='count(h:AttachedToCoolingSystem) = 1'>Expected 1 element(s) for xpath: AttachedToCoolingSystem</sch:assert>
+      <sch:assert role='ERROR' test='count(../h:CoolingSystem/h:CoolingSystemType[text()="packaged terminal air conditioner"]) &gt;= 1'>Expected 1 or more element(s) for xpath: ../CoolingSystem/CoolingSystemType[text()="packaged terminal air conditioner"]</sch:assert>
```

Comment:
Nice check on cooling system type.

Reply:
I suggest exposing the heating efficiency (%) as an input rather than hard-coding it in hvac.rb.

Reply:
I think it is ended up like this because we're not sure how the ptac heating efficiency will be rated (eg. unit % or afue). Do you suggest we could simply accept %  or both (regarding to afue and % are modeled in the same way)?

Reply:
I guess we can proceed with % since we only support electric resistance as ptac heating for now.

Reply:
> I guess we can proceed with % since we only support electric resistance as ptac heating for now.

👍 That was going to be my suggestion.